src/libasr/codegen/wasm_assembler.h

```diff
@@ -904,345 +243,6 @@ void save_bin(Vec<uint8_t> &code, std::string filename) {
-void emit_i64_load32_s(Vec<uint8_t> &code, Allocator &al, uint32_t mem_align,
-                       uint32_t mem_offset) {
-    emit_b8(code, al, 0x34);
-    emit_u32(code, al, mem_align);
-    emit_u32(code, al, mem_offset);
-}
-
-// function to emit i64.load32_u instruction
-void emit_i64_load32_u(Vec<uint8_t> &code, Allocator &al, uint32_t mem_align,
-                       uint32_t mem_offset) {
-    emit_b8(code, al, 0x35);
-    emit_u32(code, al, mem_align);
-    emit_u32(code, al, mem_offset);
-}
-
-// function to emit i32.store instruction
-void emit_i32_store(Vec<uint8_t> &code, Allocator &al, uint32_t mem_align,
-                    uint32_t mem_offset) {
-    emit_b8(code, al, 0x36);
-    emit_u32(code, al, mem_align);
-    emit_u32(code, al, mem_offset);
-}
-
-// function to emit i64.store instruction
-void emit_i64_store(Vec<uint8_t> &code, Allocator &al, uint32_t mem_align,
-                    uint32_t mem_offset) {
-    emit_b8(code, al, 0x37);
-    emit_u32(code, al, mem_align);
-    emit_u32(code, al, mem_offset);
-}
-
-// function to emit f32.store instruction
-void emit_f32_store(Vec<uint8_t> &code, Allocator &al, uint32_t mem_align,
-                    uint32_t mem_offset) {
-    emit_b8(code, al, 0x38);
-    emit_u32(code, al, mem_align);
-    emit_u32(code, al, mem_offset);
-}
-
-// function to emit f64.store instruction
-void emit_f64_store(Vec<uint8_t> &code, Allocator &al, uint32_t mem_align,
-                    uint32_t mem_offset) {
-    emit_b8(code, al, 0x39);
-    emit_u32(code, al, mem_align);
-    emit_u32(code, al, mem_offset);
-}
-
-// function to emit i32.store8 instruction
-void emit_i32_store8(Vec<uint8_t> &code, Allocator &al, uint32_t mem_align,
-                     uint32_t mem_offset) {
-    emit_b8(code, al, 0x3A);
-    emit_u32(code, al, mem_align);
-    emit_u32(code, al, mem_offset);
-}
-
-// function to emit i32.store16 instruction
-void emit_i32_store16(Vec<uint8_t> &code, Allocator &al, uint32_t mem_align,
-                      uint32_t mem_offset) {
-    emit_b8(code, al, 0x3B);
-    emit_u32(code, al, mem_align);
-    emit_u32(code, al, mem_offset);
-}
-
-// function to emit i64.store8 instruction
-void emit_i64_store8(Vec<uint8_t> &code, Allocator &al, uint32_t mem_align,
-                     uint32_t mem_offset) {
-    emit_b8(code, al, 0x3C);
-    emit_u32(code, al, mem_align);
-    emit_u32(code, al, mem_offset);
-}
-
-// function to emit i64.store16 instruction
-void emit_i64_store16(Vec<uint8_t> &code, Allocator &al, uint32_t mem_align,
-                      uint32_t mem_offset) {
-    emit_b8(code, al, 0x3D);
-    emit_u32(code, al, mem_align);
-    emit_u32(code, al, mem_offset);
-}
-
-// function to emit i64.store32 instruction
-void emit_i64_store32(Vec<uint8_t> &code, Allocator &al, uint32_t mem_align,
-                      uint32_t mem_offset) {
-    emit_b8(code, al, 0x3E);
-    emit_u32(code, al, mem_align);
-    emit_u32(code, al, mem_offset);
-}
```

Comment:
All these are now generated automatically from `wasm_intructions_visitor.py`?

Reply:
Yes, they are automatically generated.

Reply:
That's excellent. Much less code to maintain.